index.js

```diff
@@ -35,7 +35,9 @@ for (var i = 0; i < filesLoadedBeforeTrace.length; i++) {
   var matches = moduleRegex.exec(filesLoadedBeforeTrace[i]);
   if (matches && matches.length > 1 &&
       modulesLoadedBeforeTrace.indexOf(matches[1]) === -1) {
-    modulesLoadedBeforeTrace.push(matches[1]);
+    if (matches[1] !== '@google' && matches[1] !=='cloud-trace') {
```

Comment:
Thanks for submitting this PR. Have you observed `'cloud-trace'` appearing in the warning on startup? The `'@google'` check will eliminate the incorrect warning but I'm not seeing a code path where `'cloud-trace'` could appear as well.